Scripts/modelsystem.py

```diff
@@ -113,6 +114,7 @@ def _add_internal_demand(self, previous_iter_impedance, is_last_iteration):
         self.dm.generate_tours()
         
         # Assigning of tours to mode, destination and time period
+        self.passenger_modes = set()
```

Comment:
Rather `transport_modes` or `travel_modes`

Reply:
Yep! I was thinking that there must be better name for this variable, but somehow couldn't figure out what it should be.